apis/python/notebooks/tutorial_soma_objects.ipynb

```diff
@@ -0,0 +1,1442 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "ab458224-5353-4e15-baa9-46689729e071",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import tiledbsoma"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "20d435ab-7020-4e11-a71f-ac9b734f674a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from tiledbsoma.io.ingest import from_h5ad"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "29d2ea30-47b7-4e78-a8af-da66b90124b2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/opt/TileDB-SOMA/doc/venv/lib/python3.10/site-packages/anndata/compat/__init__.py:232: FutureWarning: Moving element from .uns['neighbors']['distances'] to .obsp['distances'].\n",
+      "\n",
+      "This is where adjacency matrices should go now.\n",
+      "  warn(\n",
+      "/opt/TileDB-SOMA/doc/venv/lib/python3.10/site-packages/anndata/compat/__init__.py:232: FutureWarning: Moving element from .uns['neighbors']['connectivities'] to .obsp['connectivities'].\n",
+      "\n",
+      "This is where adjacency matrices should go now.\n",
+      "  warn(\n"
+     ]
+    },
+    {
+     "ename": "AttributeError",
+     "evalue": "'str' object has no attribute 'close'",
```

Comment:
Needs a rebase to `main` as we have merged #1053, since the last rc, for improved UX.

(Also: I'd like to tag another rc after #1061 is merged -- including #1053 -- as it will make everyone's notebook work easier.)

Reply:
Will do! Also, these cells were meant for generating the fixture from an H5AD. Since I committed the SOMA-ized fixture, I think these don't belong to this notebook anymore.